python_modules/dagster-graphql/dagster_graphql/implementation/fetch_asset_condition_evaluations.py

```diff
@@ -45,12 +44,11 @@ def _get_graphene_records_from_evaluations(
 
     partitions_defs = {}
 
-    nodes = get_asset_nodes_by_asset_key(graphene_info)
     for asset_key in asset_keys:
-        asset_node = nodes.get(asset_key)
+        remote_node = graphene_info.context.asset_graph.get(asset_key)
```

Comment:
`get` throws on miss and this existing code handles missing keys so need a `has` check here first